### ISSUE TITLE
Relative paths in `style_sheet!` macro

### ISSUE THREAD

**Issue by SSeanin** [myFavShrimp/turf#13]
Is there any plan to allow for relative paths in macros such as `style_sheet!`?

**Comment by myFavShrimp** [comment]
Yes this is planned and support for relative file paths will be added when it is possible to obtain a [`SourceFile`'s](https://doc.rust-lang.org/proc_macro/struct.SourceFile.html) path on stable rust (see https://github.com/rust-lang/rust/issues/54725)

**Comment by SSeanin** [comment]
Thanks @myFavShrimp.
